mgmtsystem/openstack.py

```diff
@@ -438,6 +438,7 @@ def deploy_template(self, template, *args, **kwargs):
         Args:
             template: The name of the template to use.
             flavour_name: The name of the flavour to use.
+            flavour_id: UUID of the flavour to use.
```

Comment:
You either use name (the current way) or you an use UUID, which is more reliable. I don't want to use name in my upcoming sprout PR so I require UUID. I want to be explicit.